test/Microsoft.OpenAPI.OData.Reader.Tests/Edm/ODataPathProviderTests.cs

```diff
@@ -617,6 +618,89 @@ public void GetPathsWithStreamPropertyAndWithEntityHasStreamWorks(bool hasStream
             }
         }
 
+        [Fact]
+        public void GetPathsWithAlternateKeyParametersWorks()
+        {
+            string alternateKeyProperty =
+@"<Property Name=""SSN"" Type=""Edm.String""/>
+    <Annotation Term=""OData.Community.Keys.V1.AlternateKeys"">
```

Comment:
ODL supports to use OData.Core.Vocabulary alternateKeys. And that's the recommended solution.

Consider to enable this.

Reply:
For some reason graph metadata uses `OData.Community.Keys.V1.AlternateKey`. I'm changing the test to use `Org.OData.Core.V1.AlternateKey`.